### PR TITLE
Adding field "content" to the TrafficSign

### PR DESCRIPTION
**Pull request by katarinagresova**
Field 'content' will be filled with the text extracted from the traffic sign.

### CODE REVIEW

File: semantic/io.cpp [hunk 1]
```diff
@@ -343,6 +343,8 @@ void parse(TrafficSign &trafficSign, const Json::Value &value)
     parse(trafficSign.views, Json::check(value, "views", Json::arrayValue));
     parse(trafficSign.size, Json::check(value, "size", Json::arrayValue));
     Json::get(trafficSign.className, value, "className");
+    Json::get(trafficSign.content, value, "content");
+    Json::get(trafficSign.classId, value, "classId");
```
Comment (MartinaBekrova) [inline semantic/io.cpp:346]: Do not include changes from Tomas, you only want to add your attribute.

File: semantic/io.cpp [hunk 2]
```diff
@@ -343,6 +343,8 @@ void parse(TrafficSign &trafficSign, const Json::Value &value)
     parse(trafficSign.views, Json::check(value, "views", Json::arrayValue));
     parse(trafficSign.size, Json::check(value, "size", Json::arrayValue));
     Json::get(trafficSign.className, value, "className");
+    Json::get(trafficSign.content, value, "content");
```
Comment (MartinaBekrova) [inline semantic/io.cpp]: I think that better name of this attribute would be "text".

Comment (katarinagresova) [reply semantic/io.cpp]: Agree, done.

File: semantic/io.cpp (hunk 1 above)
Comment (katarinagresova) [reply semantic/io.cpp:346]: Resolved after merging changes from Tomas.

**Pull request merged**